spv/sync.go

```diff
@@ -12,6 +12,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	"decred.org/dcrwallet/v4/chain"
```

Comment:
i realize this is a draft, but don't depend on this package (poorly named, but is the rpc-mode syncer) from the spv syncer.

Reply:
moving the correct code into (*Wallet).SetBirthState seems more appropriate.

Reply:
Moved to to the `wallet` package https://github.com/decred/dcrwallet/compare/d54c90be3f60e1b0dc3bcb996c05d03f87a74ce9..0401f055e579da3689800add77abf42392584300